controllers/managedfusionoffering_controller.go

```diff
@@ -358,10 +358,16 @@ func pluginGetDesiredSubscriptionSpec(r *ManagedFusionOfferingReconciler) *opv1a
 			Channel: "stable-4.12",
 			Package: "ocs-operator",
 			Config: opv1a1.SubscriptionConfig{
-				Env: []corev1.EnvVar{{
-					Name:  "SKIP_NOOBAA_CRD_WATCH",
-					Value: "true",
-				}},
+				Env: []corev1.EnvVar{
+					{
+						Name:  "SKIP_NOOBAA_CRD_WATCH",
+						Value: "true",
+					},
+					{
+						Name:  "ROOK_DISABLE_OBJECT_BUCKET_CLAIM",
```

Comment:
- testing my understanding, these env values will be added to ocs-operator and that operator will make corresponding decision, isn't it?

Reply:
These env var will get added to all deployments mentioned in the ocs operator CSV. One of the deployment is rook operator and rook will skip watching OB and OBC CRs.

Reply:
ack, so it's rook then.

Reply:
`ROOK_DISABLE_OBJECT_BUCKET_CLAIM` is used by rook to skip OB and OBC
`SKIP_NOOBAA_CRD_WATCH` is used by ocs-operator to skip NooBaa